integration-tests/__tests__/wallet/transfer-ticket-operation.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { CONFIGS } from '../../config';
+import { DefaultContractType, TezosToolkit } from '@taquito/taquito';
+import { ticketsSendTz } from '../../data/code_with_ticket_transfer';
+import { RpcClient, TicketTokenParams } from '@taquito/rpc';
+
+CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
+  const Tezos = lib;
+  const client = new RpcClient(rpc);
+
+  let ticketSendContract: DefaultContractType;
+  let recipient: TezosToolkit;
+  let sender: TezosToolkit;
+  let recipientPkh: string;
+  let senderPkh: string
+  let ticketToken: TicketTokenParams;
+
+  describe(`Transfer tickets between implicit accounts using: ${rpc}`, () => {
+
+    beforeAll(async () => {
+      await setup(true);
+      try {
+        recipient = await createAddress();
+        sender = await createAddress();
+
+        recipientPkh = await recipient.signer.publicKeyHash();
+        senderPkh = await sender.wallet.pkh();
+
+        const fundSender = await Tezos.contract.transfer({ to: senderPkh, amount: 5 });
+        await fundSender.confirmation();
+
+        const ticketSendOrigination = await Tezos.contract.originate({ code: ticketsSendTz, storage: null });
```

Comment:
Seeing ticketsSendTz originated here the 2nd time made me want to originate it at `integration-tests/originate-known-contracts.ts` as knownTicketContract. And `integration-tests/__tests__/rpc/nodes.spec.ts` can benefit from this too.

Reply:
that's not a bad idea, we can make a ticket for that.